vireo/ltmain.sh

```diff
@@ -1649,7 +1649,7 @@ func_options ()
 # All initialisations required before starting the option parse loop.
 # Note that when calling hook functions, we pass through the list of
 # positional parameters.  If a hook function modifies that list, and
-# needs to propogate that back to rest of this script, then the complete
+# needs to propagate that back to rest of this script, then the complete
```

Comment:
this is an auto generated file and the typo is coming from a tool. it will be overwritten, so let's remove this from the PR.